### ISSUE TITLE
getNeighbors address field always displays FQDN

### ISSUE THREAD

**Issue by nuriel77** [iotaledger/hornet#118]
**Describe the bug**
- Add a neighbor via FQDN (a resolvable one)
- Once you call the API (getNeighbors) both the "address" and "domain" fields show the FQDN
In IRI, when I add a neighbor via FQDN, the address field always shows the IP address this FQDN resolves to.

**Expected behavior**
If Hornet's API is to be compatible with IRI's, I'd expect the "address" field to show the IP address of the neighbor's FQDN and not the FQDN itself.

**Environment information:**
 - HORNET 0.2.12

**Comment by TeeVeeEss** [comment]
I agree: ``getNeighbors`` should always show the notation of neighbors how they are defined in the config when the node starts. In IRI that's always shown in the field "domain". So if you define one nbr as tcp://11.22.33.444:15600 and another as tcp://example.com:15600, then "domain" will show "11.22.33.444" and "example.com".
The field "address" will show "11.22.33.444:15600" and "[ip-of-example.com]:15600", as IRI resolves all FQDN's to their ip-address. This is needed for nodes who are on a dynamic ip and use services like dyndns to have a static FQDN to be given to their neighbors.
In short: "domain" should always show the initial neighbors notation, "address" the resolved ip-addresses of that notation.
